### PR TITLE
Updated BpkBadge design.

### PR DESCRIPTION
**Pull request by bvitaliyg**
+ [ ] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-android/blob/main/CONTRIBUTING.md)

Remember to include the following changes:
+ [x] `UNRELEASED.md`
+ [x] `README.md`
+ [x] Tests
+ [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/main/CODE_REVIEW_GUIDELINES.md)_

### CODE REVIEW

File: Backpack/src/internal/res/values/backpack.font.xml [hunk 1]
```diff
@@ -11,8 +11,6 @@
 
     <item name="android:lineSpacingMultiplier">1</item>
 
-    <item name="bpkBadgeStyle">@style/InternalBpkBadge</item>
```
Comment (bvitaliyg) [inline Backpack/src/internal/res/values/backpack.font.xml:14]: It was only use for includeFontPadding attribute.

File: Backpack/src/main/java/net/skyscanner/backpack/badge/BpkBadge.kt [hunk 2]
```diff
@@ -29,13 +30,12 @@ import androidx.annotation.ColorRes
 import androidx.core.content.ContextCompat
 import net.skyscanner.backpack.R
 import net.skyscanner.backpack.text.BpkText
-import net.skyscanner.backpack.util.createContextThemeWrapper
 
 open class BpkBadge @JvmOverloads constructor(
   context: Context,
   attrs: AttributeSet? = null,
   defStyleAttr: Int = 0
-) : BpkText(createContextThemeWrapper(context, attrs, R.attr.bpkBadgeStyle), attrs, defStyleAttr) {
```
Comment (bvitaliyg) [inline Backpack/src/main/java/net/skyscanner/backpack/badge/BpkBadge.kt:38]: We don't use theme attribute anymore.

File: Backpack/src/main/java/net/skyscanner/backpack/badge/BpkBadge.kt [hunk 3]
```diff
@@ -52,23 +52,23 @@ open class BpkBadge @JvmOverloads constructor(
     /**
      * Style for badges with positive messages
      */
-    Success(1, R.color.bpkGlencoe, R.color.bpkSkyGrayTint01),
```
Comment (bvitaliyg) [inline Backpack/src/main/java/net/skyscanner/backpack/badge/BpkBadge.kt:55]: Updated the colours according to the design.

File: app/src/main/res/values/themes.xml [hunk 4]
```diff
@@ -77,10 +76,6 @@
     <item name="chipSelectedBackgroundColor">?bpkPrimaryColor</item>
   </style>
 
-  <style name="LondonThemeBadge">
```
Comment (bvitaliyg) [inline app/src/main/res/values/themes.xml:80]: This is used in our sample app. No need to use it anymore.

**Review by shaundon — APPROVED** [review]

**Comment by shaundon** [comment]
I didn't run it locally, but based on the diff and screenshots it all looks good

**Pull request merged**